libpermian/plugins/run_subset/__init__.py

```diff
@@ -180,6 +198,7 @@ def subset_command(base_parser, args):
             'testcases_queries': options.testcase_query,
             'configurations': options.configuration,
             'crc_queries': options.crc_query,
+            'display_name': options.display_name
```

Comment:
Please add ',' at the end of the line so that this line doesn't need to be modified if there's anything else added later.